stop_redact.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 set -o errexit
-set -o nounset
 set -a
-source docker-compose.env
+installation_dir=${INSTALLATION_DIR}
```

Comment:
I've changed it now to take the full installation directory as an argument, so this should be solved. I also added a note to discourage the use of stop/start_redact.sh after installing the service, since once it's installed as a service, most interactions should be done via systemctl.